datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java

```diff
@@ -343,7 +344,11 @@ private String getFailureReason(Map<Object, Object> variables, Exception excepti
         if (variables.containsKey(REASON) && variables.get(REASON).equals(DatalakeBackupFailureReason.BACKUP_ON_UPGRADE.name())) {
             reason.append("Upgrade not started, datalake backup failed.");
         } else {
-            reason.append("Backup failed, returning datalake to running state.");
+            if (exception instanceof PollerStoppedException) {
+                reason.append("Backup timed out, see backup status for latest.");
```

Comment:
How about, "Backup timed out, see the backup status using cdp-cli for the latest".